source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/display.tsx

```diff
@@ -142,7 +96,48 @@ class Display extends React.Component<IProps> {
           </div>
         </div>
 
-        <Tab menu={{ text: true }} panes={this.renderTabPanes()} className='tabs' />
+        <Tab
```

Comment:
this didn't _need_ to happen, but it kept confusing me that I had to scroll _up_ to something that has yet to be rendered